adr/0000-use-OWNERS-rather-than-github-teams.md

```diff
@@ -0,0 +1,37 @@
+# Use OWNERS to determine roles in a project/repository
+
+* Status: proposed
+
+Technical Story: we want to have approvers and reviewers for every repository we maintain on GitHub
+
+## Context and Problem Statement
+
+As we want to conduct code reviews, we want to implemente a reviewer-approver pattern (as seen in Kubernetes, Operate-First, or Thoth Station),
+therefore we need to maintain theres roles on GitHub. As Prow (specifically peribolos) is our janitor, we need to declare
+the roles within each repository and/or within the organization.
+
+## Decision Drivers <!-- optional -->
+
+* clarity and easy tracability
+
+## Considered Options
+
+* OWNERS-teams-mix - we can use the OWNERS file in each repository to declare reviews/approver roles by referencing a GitHub team
+* OWNERS-only - we can use the OWNERS file to explicitly declare the roles
```

Comment:
Hm.. I think this option is a bit more complex than outlined here. We also use [`OWNERS_ALIASES`](https://github.com/open-services-group/community/blob/main/OWNERS_ALIASES) - as a soft drop in replacement for teams.

Reply:
I agree on this point, we should address having the OWNERS_ALIASES.

Also we should state, folks who are needed to be admin of repo(permission to change setting) should be defined in the github team and addressed with OWNERS with team.
i don't know if only stating a person as owner in OWNERS file or OWNERS_ALIASES grant them admin privileges in that repo, as my understanding it , only prow think that they are good to be maintainers and can approve pr or issues.

Reply:
... but even in the kubernetes universe, the initial commit most often comes from a human.